packages/fixtures/src/events.ts

```diff
@@ -4,49 +4,65 @@ import {
   BlockButtonAction,
   BlockElementAction,
   ButtonAction,
+  MessageEvent,
   SlashCommand,
 } from '@slack/bolt';
 
-const token = 'TOKEN';
-const response_url = 'https://fake.slack/response_url';
-const trigger_id = 'TRIGGER_ID';
-const api_app_id = 'API_APP_ID';
-const user = {
-  id: 'UUSERID',
-  name: 'USER',
-};
-const channel = {
-  id: 'CCHANNELID',
-  name: 'channel',
-};
-const team = {
-  id: 'TTEAMID',
-  domain: 'team-domain',
+import fields from './fields';
+
+export const message = (
+  text: string,
+  options: Partial<MessageEvent> = {},
+): MessageEvent => {
+  const { user, channel, ts } = fields;
```

Comment:
FWIW, it's important to reference this in every fn call rather than at the root b/c of the global overrides. 

I don't love this. Couldn't think of a good way around it.